informer/disk/disk.go

```diff
@@ -49,30 +65,28 @@ func (disk *Informer) Name() string {
 	return MetricName
 }
 
-// GetMetric uses the IPFSConnector the current
-// repository size and returns it in a metric.
 func (disk *Informer) GetMetric() api.Metric {
 	if disk.rpcClient == nil {
 		return api.Metric{
 			Valid: false,
 		}
 	}
 
-	var repoSize int
+	var metric int
 	valid := true
 	err := disk.rpcClient.Call("",
 		"Cluster",
-		"IPFSRepoSize",
+		nameToRPC[MetricName],
```

Comment:
either handle the metricName -> rpcname conversion via a switch statement (doing everything here in the function) with a `default` section or add code to make sure that the metric name is in the nameToRPC map. If not, log the problem clearly and return an invalid metric.

Right now the error logged would probably be very cryptic (rpc method missing i guess) and probably difficult to figure out it's just from a typo in the metric name.